Documentation/setup.txt

```diff
@@ -0,0 +1,78 @@
+EKS Factory Reference Guide
```

Comment:
@parkand1 would you mind getting this into Markdown? Also, should we consider leveraging the repo Wiki and moving this there? I imagine we can leverage the wiki as the documentation grows.

Reply:
Yes, will do